netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
 |9.3.0 |10.0.0
 |===
 
-== Changes in This Release
+== New Features and Changes in This Release
 
-This release contains the following changes:
+This release contains the following new features:
 
-* These authentication types were removed:
-** *Login Authentication* 
-** *SSO Authentication* 
 * These operations were added: 
 ** *Get Account Governance Info* 
 ** *Get Integration Governance Info* 
+
+
+== Changed Functionality in This Release
+
+This release contains the following changed functionality:
+
+* The Login and SSO authentication types were removed. +
+If your integration is based on these authentication types, you must migrate to a Request or Token-Based authentication type.
+* Other nonfunctional changes were applied to the interface of the connector to make it more consistent with other Anypoint Connectors. +
+This means that some of the operation calls in your existing application might not initially work until some minor migration tasks are performed.
+* Unknown errors are now thrown as `ANY` rather than `UNKNOWN`. +
+Consequently, error codes `DISPOSE_ERROR`, `MAPPING_ERROR`, `CANT_READ_FILE`, and `TRIGGER` are now considered `ERROR_FAULT` errors instead of `UNKNOWN`.
```

Comment:
I made edits to these wordings in the latest PR I edited for you...I believe it was Release notes for NetSuite Connector 10.0.0 #2474?
If you implemented those rewordings, please make these consistent with those.

Reply:
This actually **is** your rewording from the release notes.   So ??

Reply:
Oh! I thought that I made changes like these:

* The Login and SSO authentication types are replaced by Request and Token-Based authentication
(or You must migrate your Login and SSO types to ....)

* Other nonfunctional changes to the connector interface might not initially work, until migration is complete.

* Unknown errors (`UNKNOWN`) are thrown as `ANY` and considered `ERROR_FAULT` errors rather than `UNKNOWN`.

I was looking for single-sentence bullets.
But what is here is fine. Whichever you prefer or think is clearest. Wasn't sure the audience needed the detail, which I expect in the doc rather than the notes.